tools/src/main/java/com/graphhopper/tools/Measurement.java

```diff
@@ -200,26 +204,31 @@ protected DataReader importData() throws IOException {
                 if (!blockAreaStr.isEmpty())
                     printTimeOfRouteQuery(hopper, new QuerySettings("routing_block_area", count / 20, isCH, isLM).
                             withInstructions().blockArea(blockAreaStr));
+                // todonow: use custom model as query custom model (if given)
+                // todonow: do we really need to test this here?
+//                printTimeOfRouteQuery(hopper, new QuerySettings("routing_custom_truck", count / 20, isCH, isLM).
+//                        profile("custom_truck").withInstructions().queryCustomModel(createCustomProfile().getCustomModel()));
```

Comment:
Do you think this is important for the measurement? For performance there should be no difference between using a custom model per query and using the one configured on the server-side? Or were you worried about the performance of merging the models or something?

Reply:
Yes, my fear is that the merging does something wrong causing perf problems. But we can remove it for now.

Reply:
If merging takes to long maybe write a test that does only this?

Reply:
I don't mean that merging takes long but that merging is done wrong so that the speed is changed a lot. Probably these bugs are only visible over a longer time, not sure if they could be catched with this...

Reply:
Ah I see. Hmm. If its about correctness maybe its better to do it in some test? But if you think its important feel free to add it again. I just removed it because I was trying to make the Measurement class unaware of the custom models it is used with.